201-userdefined-routes-appliance/azuredeploy.json

```diff
@@ -0,0 +1,553 @@
+				"description": "The Windows version for the VM. This will pick a fully patched image of this given Windows version. Allowed values: 2008-R2-SP1, 2012-Datacenter, 2012-R2-Datacenter."
+			}
+		},
+		"PublicIPNameForVM1": {
+			"type": "string",
+			"defaultValue": "FEVM1_PIP",
+			"metadata": {
+				"description" : "Name of the Public IP"
+			}
+		},
+		"PublicIPNameForVM2": {
+			"type": "string",
+			"defaultValue": "NVAVM1_PIP",
+			"metadata": {
+				"description" : "Name of the Public IP"
+			}
+		},
+		"PublicIPNameForVM3": {
+			"type": "string",
+			"defaultValue": "BEVM1_PIP",
+			"metadata": {
+				"description" : "Name of the Public IP"
+			}
+		},
+		"VNetName": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the VNet"
+			}	
+		},
+		"Subnet1Name": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the Subnet"
+			}	
+		},
+		"Subnet2Name": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the Subnet"
+			}	
+		},
+		"Subnet3Name": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the Subnet"
+			}	
+		},
+		"routeTableName": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the Route Table"
+			}	
+		},
+		"VM1Name": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the VM"
+			}	
+		},
+		"VM2Name": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the VM"
+			}	
+		},
+		"VM3Name": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "Name of the VM"
+			}	
+		},
+		"NVAPrivateIP": {
+			"type": "string",
+			"defaultValue": "",
+			"metadata": {
+				"description" : "IP Address of the NVA"
+			}	
+		}
+		},
+
+	 "variables" : {
+		"imagePublisher": "MicrosoftWindowsServer",
+		"imageOffer": "WindowsServer",
+		"OSDiskName": "osdiskforwindowssimple",
+		"storageAccountType": "Standard_LRS",
+		"vmStorageAccountContainerName": "vhds",
+        "vmName1": "[parameters('VM1Name')]",
+		"vmName2": "[parameters('VM2Name')]",
+		"vmName3": "[parameters('VM3Name')]",
+        "vmSize": "Standard_A2",
+		"vnetID":"[resourceId('Microsoft.Network/virtualNetworks', parameters('VNetName'))]",
+		"subnet1Ref" : "[concat(variables('vnetID'),'/subnets/', parameters('Subnet1Name'))]",
+		"subnet2Ref" : "[concat(variables('vnetID'),'/subnets/', parameters('Subnet2Name'))]",
+		"subnet3Ref" : "[concat(variables('vnetID'),'/subnets/', parameters('Subnet3Name'))]",
+		"routeTableName" : "[parameters('routeTableName')]",
+		"NvmPrivateIPAddress" : "[parameters('NVAPrivateIP')]",
+		"nsgname": "DefaultNSG"
+    },
+    "resources": [
+	 {
+		"type": "Microsoft.Storage/storageAccounts",
+        "name": "[parameters('newStorageAccountName')]",
+        "apiVersion": "2015-05-01-preview",
```

Comment:
Can you make api version a variable which will make it easier to move to newer API versions